mitodl/mit-open
# Change channel type and url from "offeror" to "unit"

### What are the relevant tickets?
Closes https://github.com/mitodl/hq/issues/4508

### Description (What does it do?)
Changes the channel_type value of offeror to unit, the url paths of offeror channels from /c/offeror/ to /c/unit. and renames many clases from offeror to unit.


### How can this be tested?
- If you haven't created any channels yet, run the following on t he `main` branch: `./manage.py backpopulate_resource_channels`
- Switch to this branch and restart containers.  Migrations should run without errors.
- Go to `http://localhost:8063/c/unit/mitx/` - the page should load without errors
